dockstore-integration-testing/src/test/java/io/dockstore/client/cli/NextflowHandlerIT.java

```diff
@@ -0,0 +1,120 @@
+package io.dockstore.client.cli;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import io.dockstore.common.CommonTestUtilities;
+import io.dockstore.common.DescriptorLanguage;
+import io.dockstore.webservice.core.SourceFile;
+import io.dockstore.webservice.core.WorkflowVersion;
+import io.dockstore.webservice.helpers.SourceCodeRepoFactory;
+import io.dockstore.webservice.helpers.SourceCodeRepoInterface;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ *  TODO: Use fork with tagged version
+ *  TODO: Check paths, something's not right
+ */
+public class NextflowHandlerIT extends BaseIT {
+    protected static SourceCodeRepoInterface sourceCodeRepoInterface;
+
+
+    @Before
+    @Override
+    public void resetDBBetweenTests() throws Exception {
+        CommonTestUtilities.addAdditionalToolsWithPrivate2(SUPPORT, false);
+        String githubToken = testingPostgres
+                .runSelectStatement("select content from token where username='DockstoreTestUser2' and tokensource='github.com'",
+                        String.class);
+        sourceCodeRepoInterface = SourceCodeRepoFactory.createGitHubAppRepo(githubToken);
+    }
+
+    /**
+     * Tests:
+     * Nextflow DSL2 with imports that import
+     */
+    @Test
+    public void testProcessImportsRnaseq() {
+        final String githubRepository = "nextflow-io/rnaseq-nf";
```

Comment:
will these tests break if `nextflow-io` changes things (or are these forked and I'm missing it?)

Reply:
Yup, going to fork and create tag before merge